dom.go

```diff
@@ -750,6 +751,10 @@ func (d document) DocumentURI() string {
 	return d.Get("documentURI").String()
 }
 
+func (d document) ExecCommand(name string, showDefaultUI bool, valueArgument string) bool {
+	return d.Call("execCommand", name, showDefaultUI, valueArgument).Bool()
+}
```

Comment:
I'm not surprised. Most JavaScript APIs will "work" when given input other than their documentation says they expect.

But I still think we should follow the documentation precisely here. Otherwise, people who look at this Go API will have questions raised in their minds, like "does this actually work despite not matching what the JS API expects?"